include/ck_tile/ops/grouped_convolution/utils/transform_conv_fwd_to_gemm.hpp

```diff
@@ -1510,6 +1436,282 @@ struct TransformConvFwdToGemm
         }
     }
 
+    // ═══════════════════════════════════════════════════════════════════════
+    // Split-Image Calculation (AFTER Split-N)
+    // ═══════════════════════════════════════════════════════════════════════
+    // This method calculates split-image information using N_ (after Split-N).
+    // This ensures correct offset calculations when both Split-N and Split-Image
+    // are active simultaneously.
+
+    public:
+    CK_TILE_HOST SplitImageInfo<IndexType> CalculateSplitImage() const
+    {
+        SplitImageInfo<IndexType> info;
+        info.should_split = false;
+
+        // Use unified threshold from TwoGB constant
+        const long_index_t threshold_elements = TwoGB / sizeof(CDataType);
+
+        // Determine which dimension to split based on NDimSpatial
+        IndexType out_total, in_total, left_pad, right_pad, stride, dilation, filter;
+
+        if constexpr(NDimSpatial == 1)
+        {
+            out_total = Wo_;
+            in_total  = Wi_;
+            left_pad  = InLeftPadW_;
+            right_pad = InRightPadW_;
+            stride    = ConvStrideW_;
+            dilation  = ConvDilationW_;
+            filter    = X_;
+        }
+        else if constexpr(NDimSpatial == 2)
+        {
+            out_total = Ho_;
+            in_total  = Hi_;
+            left_pad  = InLeftPadH_;
+            right_pad = InRightPadH_;
+            stride    = ConvStrideH_;
+            dilation  = ConvDilationH_;
+            filter    = Y_;
+        }
+        else if constexpr(NDimSpatial == 3)
+        {
+            out_total = Do_;
+            in_total  = Di_;
+            left_pad  = InLeftPadD_;
+            right_pad = InRightPadD_;
+            stride    = ConvStrideD_;
+            dilation  = ConvDilationD_;
+            filter    = Z_;
+        }
```

Comment:
In previous implementation we were able to split per each spatial dimension for examples
H = 2, W =64 tensor size = 8 GB
Than we can split H to 1 and W to 32. In this one it is not possible